packages/docs/docs/getting-started/create-unirep-app.md

```diff
@@ -159,114 +142,129 @@ Requesting data with epoch key:
+The example input fields take the data a user wishes to claim. Clicking the **Generate Proof** button creates a data proof with the claim data and sends it to the prover to be verified.<br/>
+A proof will be valid if the claimed data is greater than or equal to the user's provable data.<br/>
 
+**e.g.**<br/>
 If user's provable data is:
 ```shell
 Provable Data 0 = 2
 Provable Data 1 = 3
 Provable Data 2 = 4
 Provable Data 3 = 5
 ```
-and user can claim he has data:
+the user can claim they have data:
 ```shell
 Claim Data 0 = 2
 Claim Data 1 = 2
 Claim Data 2 = 2
 Claim Data 3 = 2
 ```
 
-since:
+because:
 ```shell
 2 (Provable Data 0) >= 2 (Claim Data 0) # satisfied. 2 === 2
 3 (Provable Data 1) >= 2 (Claim Data 1) # satisfied. 3 > 2
 4 (Provable Data 2) >= 2 (Claim Data 2) # satisfied. 4 > 2
 5 (Provable Data 3) >= 2 (Claim Data 3) # satisfied. 5 > 2
 ```
 
-If the proof is valid, it shows the proof and "Is proof valid? **true**" below the 'Generate Proof' button. <br/>
-If the proof is invalid, the snarkjs prover throws an error.
-
-e.g.
+If the proof is valid, the frontend will display the proof and "Is proof valid? **true**" will read below the **Generate Proof** button. <br/>
+If the proof is invalid, the snarkjs prover will throw an error.
 
+**e.g.**<br/>
 If user's provable data is:
 ```shell
 Provable Data 0 = 2 👈
 Provable Data 1 = 3
 Provable Data 2 = 4
 Provable Data 3 = 5
 ```
-and user claims he has data:
+and they claim to own the data:
 ```shell
 Claim Data 0 = 3 👈
 Claim Data 1 = 3
 Claim Data 2 = 3
 Claim Data 3 = 3
 ```
 
-It throws an error since the fields where the emojis are do not satisfy the conditions:
+The prover will throw an error because the fields with emojis do not satisfy the conditions:
 
 ```shell
 2 (Provable Data 0) >= 3 (Claim Data 0) # not satisfied. 2 is not greater than or equal to 3
 ```
 
+:::info
+See the customized [data proof circuit](https://github.com/Unirep/create-unirep-app/blob/2ca9aaa3fcacb9282993b8f5d5917afc482ec089/packages/circuits/circuits/dataProof.circom)<br/>
+App builders can customize their own ZK circuits and deploy verifiers to fit the application.
+:::
+
 ## 💡 Build your own application
 
-Start with modifying the [`UnirepApp.sol`](https://github.com/Unirep/create-unirep-app/blob/2ca9aaa3fcacb9282993b8f5d5917afc482ec089/packages/contracts/contracts/UnirepApp.sol).
+Start by modifying the [`UnirepApp.sol`](https://github.com/Unirep/create-unirep-app/blob/2ca9aaa3fcacb9282993b8f5d5917afc482ec089/packages/contracts/contracts/UnirepApp.sol) contract.
 
 :::info
 See all [`Unirep.sol` APIs](../contracts-api/unirep-sol.md)
 :::
 
-### Change epoch length
+### Sign up users
+
+Attesters may allow anyone to join and use their application, as in the example attester, or..
+
+e.g:
+
+<!-- TODO:
+provide ideas how an attester can require zk proof for users to join, example from zketh? -->
```

Comment:
@vivianjeng should we include this section? I think for 'getting started' we should try to be comprehensive, covering each concept.. but not to go into explanations that are  unnecessary, what do you think?

i've thought it would be good to add another section at the end to deal with proofs - some discussion / examples about how an attester can make use of them with their application logic. Do you think this would helpful for (maybe less experienced?)  devs in getting a grasp of what/how to build around the protocol functionality, or its it too obvious that it shouldn't be included?

Reply:
I think Chiali is going to include the examples in introduction page
https://github.com/Unirep/Unirep/blob/2b22cc76d481056e2363cb1318907c342ee72b21/packages/docs/versioned_docs/version-2.0.0-beta-3/welcome.md
(But hasn't been merged yet)